x-pack/test/examples/config.ts

```diff
@@ -26,7 +26,6 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
     testFiles: [
       require.resolve('./search_examples'),
       require.resolve('./embedded_lens'),
-      require.resolve('./reporting_examples'),
```

Comment:
I'm in favor of removing the examples, since the examples were used to demonstrate stateful components that are given from the Reporting plugin contract, and that's not really the type of integration that we want to promote any more.